src/privatesend/privatesend-client.h

```diff
@@ -7,8 +7,6 @@
 
 #include <privatesend/privatesend-util.h>
 #include <privatesend/privatesend.h>
-#include <wallet/wallet.h>
```

Comment:
e3f0108d7bae13969d90094e5f85ee6a02b0f607 message probably shouldn't have prefix exclusively be `qt`

Reply:
See e031596